src/components/results/dynamicsimulation/dynamic-simulation-result-time-series.tsx

```diff
@@ -123,61 +124,65 @@ const DynamicSimulationResultTimeSeries = memo(function ({
                 </Box>
             )}
             <Overlay message={overlayMessage}>
-                <Box sx={styles.root}>
-                    <Box sx={{ display: 'flex', flexDirection: 'row' }}>
-                        {/* tab headers */}
-                        <DroppableTabs
-                            id={'1'}
-                            value={selectedIndex}
-                            onChange={handleTabsChange}
-                            tabsRender={() =>
-                                tabs.map((tab, index) => {
-                                    return (
-                                        <DraggableTab
-                                            key={`tab-${tab.id}`}
-                                            id={`tab-${index}`}
-                                            index={index}
-                                            value={index}
-                                            label={
-                                                <span
-                                                    style={{
-                                                        whiteSpace: 'nowrap',
-                                                    }}
-                                                >
-                                                    {`${intl.formatMessage({
-                                                        id: 'DynamicSimulationResultTab',
-                                                    })} ${tab.id}`}
-                                                    <TooltipIconButton
-                                                        tooltip={intl.formatMessage({
-                                                            id: 'DynamicSimulationCloseTab',
-                                                        })}
-                                                        size="small"
-                                                        component="span"
-                                                        onClick={handleClose(index)}
+                <Grid container sx={styles.root}>
```

Comment:
Grid configured with container make the layout does not work, i will revert to Box

Reply:
DONE in https://github.com/gridsuite/gridstudy-app/pull/2800/commits/b3db4997a5fe1e66b71ae04f2924e317635e6208